src/Workspaces/CSharpTest/OrganizeImports/OrganizeUsingsTests.cs

```diff
@@ -1084,8 +1129,10 @@ public async Task CaseSensitivity1()
 using あｱ;
 using ああ;
 
-// If Kana is sensitive あ != ア, if Kana is insensitive あ == ア.
-// If Width is sensitiveア != ｱ, if Width is insensitive ア == ｱ.";
+// In .Net Core あ always comes before ア. In .NetFramework if Kana is sensitive あ != ア, if Kana is insensitive あ == ア.
```

Comment:
I find this comment a bit misleading, given that they only considered equal when `CompareOptions.IgnoreKanaType` is used for comparison, not `CompareOptions.IgnoreCase`, and we certainly never use the former anywhere in the source. Maybe delete it to avoid confusion

Reply:
Will delete as soon as this test run finishes